### PR TITLE
PAYARA-1926 Remove unnecessary validation for Mojarra

### PR DESCRIPTION
**Pull request by arjantijms**
PAYARA-1926 Mojarra's default was set to false some time ago, but GF and now Payara wasn't updated it seems.

### CODE REVIEW

File: nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/ResourceHandler.java [hunk 1]
```diff
@@ -40,17 +40,16 @@
...
- * ResourceHandler provides interface to access resources 
- * handled by a component.
+ * ResourceHandler provides interface to access resources handled by a component.
  */
-
 @Contract
 public interface ResourceHandler {
 
-    public java.util.List getResourceList();
+    public List getResourceList();
```
Comment (MattGill98) [inline nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/ResourceHandler.java]: That's made me feel a lot better ;)

**Comment by Pandrex247** [comment]
Jenkins, quick test Payara 5 please.

File: nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/ApplicationEnvironment.java [hunk 2]
```diff
@@ -40,19 +40,18 @@
...
 public interface ApplicationEnvironment {
     /**
-     * Returns the name of the currently running
-     * application
+     * Returns the name of the currently running application
      * 
      * @return The name of the currently running application
      */
-    public String getName();
+    String getName();
```
[the remark this answers is not in the document]
Comment (arjantijms) [reply nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/ApplicationEnvironment.java]: It's not package private, it's an interface and `public` is already the default ;)

**Comment by arjantijms** [comment]
Jenkins, quick test Payara 5 please.

File: nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/ApplicationEnvironment.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (MattGill98) [reply nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/ApplicationEnvironment.java]: This is why I'm the apprentice ;)

**Pull request merged**